File: app/src/main/java/com/liadpaz/amp/AboutActivity.java
package com.liadpaz.amp;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.liadpaz.amp.databinding.ActivityAboutBinding;

public class AboutActivity extends AppCompatActivity {
    private static final String TAG = "AmpApp.AboutActivity";

    @SuppressWarnings("ConstantConditions")
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        ActivityAboutBinding binding;
        setContentView((binding = ActivityAboutBinding.inflate(getLayoutInflater())).getRoot());
        setSupportActionBar(binding.toolbarAbout);
        getSupportActionBar().setDisplayHomeAsUpEnabled(true);

        binding.tvVersion.setText(BuildConfig.VERSION_NAME);
    }
}

File: app/src/main/java/com/liadpaz/amp/ui/AboutActivity.kt
package com.liadpaz.amp.ui

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.liadpaz.amp.BuildConfig
import com.liadpaz.amp.databinding.ActivityAboutBinding

class AboutActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        var binding: ActivityAboutBinding
        setContentView(ActivityAboutBinding.inflate(layoutInflater).also { binding = it }.root)
        setSupportActionBar(binding.toolbarAbout)
        supportActionBar!!.setDisplayHomeAsUpEnabled(true)
        binding.tvVersion.text = BuildConfig.VERSION_NAME
    }

    companion object {
        private const val TAG = "AmpApp.AboutActivity"
    }
}